feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```diff
@@ -132,6 +191,15 @@ func getSbiPduSesionType(pduSessionType protos.PduSessionType) sbi.PduSessionTyp
 	}
 }
 
+func getFailureCodeN7(failureCodeProto protos.PolicyReAuthAnswer_FailureCode) *n7_sbi.FailureCode {
+	failureCodeNum := int(failureCodeProto)
+	if failureCodeNum > len(FailureCodeProtoToN7Map) {
+		return nil
```

Comment:
doing things like that we are hiding errors. 
If for some reason we receive wrong formed value we just ignore, and don't say anything about it. 

I think some functions on `model_conversion` will need to be refactored to trigger error and stop process. It doesnt make any sense to send to AGW a proto that we know it is not formed properly.

So either PCF sends a proper message, or we error

Reply:
Please, let's create another PR to refactor moder_conversion to add error in function that deserve to error. For example this one.